bazel-mode.el

```diff
@@ -73,6 +73,20 @@
   (when bazel-mode-buildifier-before-save
     (bazel-mode-buildifier)))
 
+(defconst bazel-mode--font-lock-defaults
```

Comment:
Nit: rename this to `...-keywords` to match its usage as `font-lock-keywords`

Reply:
Done.